server/src/semantic_tokens.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as ts from 'typescript/lib/tsserverlibrary';
+import * as lsp from 'vscode-languageserver/node';
+
+export enum TokenEncodingConsts {
```

Comment:
Would you mind linking to the TS definitions like you did in https://github.com/angular/angular/pull/60260? Context is pretty lost between the two sometimes

Reply:
moved to language service